extensions/wikia/VideosModule/scripts/controllers/inContent.js

```diff
@@ -1,21 +1,27 @@
 require([
-	'videosmodule.views.index',
+	'videosmodule.views.incontent',
 	'videosmodule.models.videos',
 	'wikia.nodeFinder',
 	'wikia.mustache',
 	'videosmodule.templates.mustache',
 	'bucky',
+	'wikia.window',
 	'wikia.document',
 	'jquery'
-], function (InContentModule, VideoData, nodeFinderModule, Mustache, templates, buckyModule, doc, $) {
+], function (InContentModule,VideoData, nodeFinderModule, Mustache, templates, buckyModule, win, doc, $) {
```

Comment:
It was just a typo, but still—even if JSHint and JSCS were OK with that, according to [Mediawiki coding conventions](https://www.mediawiki.org/wiki/Manual:Coding_conventions/JavaScript#Line_length) lines should be 80-100 chars, so I'll put the params into separate lines as it also improves readability. :+1: